_plugins/jekyll-jsonld.rb

```diff
@@ -3,122 +3,324 @@
 
 module Jekyll
   module JsonldFilter
-    def to_jsonld(material)
+    def to_jsonld(material, topic, site)
+      gtn = {
+        "@type": "Organization",
+        "email": "#{site['email']}",
+        "name": "Galaxy Training Network",
+        "url": "https://galaxyproject.org/teach/gtn/"
+      }
+      topic_desc = {
+        "@type": "CreativeWork",
+        "name": "#{topic['title']}",
+        "description": "#{topic['summary']}",
+        "url": "#{site['url']}#{site['baseurl']}/topics/#{topic['name']}/"
+      }
+
+      # aggregate everything
       data = {
+        # Properties from Course
         "@context": "http://schema.org",
-        "@type": "CreativeWork",
-        "name": material['title'],
-        "accessModeSufficient": "visual",
-        "accessibilityControl": "fullMouseControl",
+        "@type": "Course",
```

Comment:
I discussed the prospect of folding the Bioschemas TrainingMaterial spec into Course; and it seems unlikely it would happen. There's lots of use-cases for non-Course Training Materials. 

Your approach of selecting whatever properties are appropriate for you across CreativeWork is good! It can be parsed by TeSS and search engines; so you'll be much more visible

Reply:
We can update later with TrainingMaterial